pkg/domain/kube/link_kube.go

```diff
@@ -0,0 +1,134 @@
+package kube
```

Comment:
Is the link abstraction the only abstraction from this domain that is implemented for kube?

Reply:
Yes, and it is an incomplete implementation of the `LinkHandler` interface.
I did it because the `cmd/skupper/skupper_link.go` now uses a `LinkHandler` implementation
in `NewCmdLinkStatus` to render the link status in a generic way.